spec/queries/fielded_search_query.rb

```diff
@@ -1,29 +1,29 @@
-describe Query::Proposal::FieldedSearch do
+describe Proposal::FieldedSearchQuery do
```

Comment:
it should! this got missed because there is no `_spec` at the end of the file -- my fault! will rename. great catch!